### ISSUE TITLE
Project Marker Disable

### ISSUE THREAD

**Issue by Simon8198** [StrangeLoopGames/EcoSuggestions#1360]
Can we get an option in the "General" Menu to turn off the automatic creation of Project Markers.

Its quite annoying having markers cluttering your map for projects that you already know where they are.

**Comment by sirPattrick** [comment]
Yes, they shouldn't be recreated every time you log in to the game.

**Comment by JestinJae** [comment]
I believe this would be well implemented as a toggle which would be saved, like the other map marker toggles.  That way it doesn't have to be a "default" that you worry about.
